gslib/tests/testcase/unit_testcase.py

```diff
@@ -169,7 +186,7 @@ def RunCommand(self, command_name, args=None, headers=None, debug=0,
 
```

Comment:
Can you indicate the type of the returned things? (i.e. is stdout and stderr going to be bytes or unicode?)

Reply:
Added comments to docstring for return values.